app/models/group.rb

```diff
@@ -1 +1,27 @@
-class Group; end
+require_relative 'student'
+
+class Group
+  attr_accessor :name,
+                :students
+
+  def initialize(name, capacity)
+    @students = []
+    @name = name
+    @capacity = capacity
+    for a in 1..capacity do
+      name = "name #{a}"
+      surname = "surname #{a}"
+      age = rand(17..24)
+      student = Student.new(name, surname, age)
+      @students.push(student)
+    end
+  end
+
+  def info
+    list = "      Group name: \"#{@name}\"\n\n"
+    list += "      Students:\n"
+    @students.sort! { |a, b|  a.name <=> b.name }
```

Comment:
in case of string you can just use @students.sort
no bang sign "!" plz
you shouldn't change students array here